vendor-builder.php

```diff
@@ -23,10 +27,20 @@
     $newComposerJson['replace'][$package['name']] = $package['version'];
 }
 
-mkdir('vendor');
+mkdir($composerWorkingDir);
 
-file_put_contents('vendor/composer.json', json_encode($newComposerJson, JSON_PRETTY_PRINT));
+file_put_contents($composerWorkingDir . '/composer.json', json_encode($newComposerJson, JSON_PRETTY_PRINT));
 
-exec('composer install -d vendor');
+exec(sprintf('COMPOSER="%s" %s install -d "%s"', $composerFile, $composerPhar, $composerWorkingDir));
```

Comment:
Whats the reason for the composer env 🤔

Reply:
It's the absolute path to the composer.json file in the plugin directory root

Reply:
All paths are now absolute, so it doesn't matter from which CWD the vendor-builder.php is called